examples/axpy_pipelined.py

```diff
@@ -61,7 +61,6 @@ def run_axpy_pipeline():
     pipeline, iterator = build_data_pipeline(input_array, map_function,
                                              batch_size)
     model = build_simple_model(pipeline)
-
     with tf.Session() as sess:
         # Initialize the globals and the dataset
         sess.run(tf.global_variables_initializer())
```

Comment:
if this script doesnt have variables we dont need this line . I removed it in my PR
sess.run(tf.global_variables_initializer())

Reply:
removed

Reply:
please remove the setting of backend interpreter in this script also